pkg/health/health.go

```diff
@@ -170,10 +214,15 @@ func findWorstResult(results []Result) Result {
 			retVal = res
 		}
 	}
-	return retVal
+	return retVal, nil
 }
 
-func findBestResult(results []Result) Result {
+func findBestResult(results []Result) (Result, *HealthEmptyError) {
+	if len(results) == 0 {
+		return Result{Status: Critical}, &HealthEmptyError{
```

Comment:
I figured passing a critical result is >= a completely empty result in terms of usefulness.